sceptre/plan/actions.py

```diff
@@ -767,6 +773,9 @@ def _protect_execution(self):
                 "currently enabled".format(self.stack.name)
             )
 
+    def _note_response_time(self, resp):
+        self.most_recent_event_datetime = get_response_datetime(resp)
```

Comment:
Technically `get_response_datetime` could return None, in which case, I don't think you want to override the existing value. 
```suggestion
        self.most_recent_event_datetime = get_response_datetime(resp) or self.most_recent_event_datetime
```

Reply:
I'm resetting it to None here so that if/when `_log_new_events` is called it will compute a default based on the system clock. But, as I mentioned above, I think there's a better way where we can eliminate this instance attribute in favor of a stack variable.